frontend/packages/kubevirt-plugin/integration-tests/views/wizard.view.ts

```diff
@@ -89,8 +89,5 @@ export const tableRowAttribute = async (name: string, columnIndex: number): Prom
     .get(columnIndex)
```

Comment:
we could move these table and kebab selectors as well

Reply:
There are very similar files like `uiResource.view` and `kubevirtUIResource.view`, I'd like to go through them in a new PR,
as this one is focusing on fixing failures.

Reply:
ok